internal/driver/gomobile/menu_test.go

```diff
@@ -5,12 +5,48 @@ package gomobile
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	internalWidget "fyne.io/fyne/internal/widget"
 	"fyne.io/fyne/theme"
 	"fyne.io/fyne/widget"
-	"github.com/stretchr/testify/assert"
 )
 
+func TestMobileCanvas_DismissBar(t *testing.T) {
+	c := NewCanvas().(*mobileCanvas)
+	c.SetContent(canvas.NewRectangle(theme.BackgroundColor()))
+	menu := fyne.NewMainMenu(
+		fyne.NewMenu("Test"))
+	c.showMenu(menu)
+	c.resize(fyne.NewSize(100, 100))
+
+	assert.NotNil(t, c.menu)
+	// simulate tap as the test util does not know about our menu...
+	c.tapDown(fyne.NewPos(80, 20), 1)
+	c.tapUp(fyne.NewPos(80, 20), 1, nil, nil, nil)
```

Comment:
What about using `test.TapCanvas`?

Reply:
This should be usable to _show_ the menu, too.

Reply:
That is what I was commenting on in the code - it does not work because gomobile driver does special things to handle menu. The TapCanvas utility only checks Content() and Overlays() trees - the menu bar is neither :(

Reply:
Ah … okay